woocommerce/sourcebuster-js
# Update project and add the `id` parameter

**This PR:**

- Includes the `utm_id` parameter
- Adds an `.nvmrc` file for Node.js v20 (lts/iron) and the `package-lock.json` file
- Updates the NPM versions and dependencies to build correctly with v20
- Swaps in `sass=dart-sass` instead of `node-sass`
- Adds new `dist` files for v1.1.2 with the above changes

**Testing**
1. Confirm that `nvm run build` works without errors after `nvm use` or `nvm install`
  - There is one gem that I had to install, too, for generating the demo files: `sudo gem install slim` 
3. Make the built `/public/`directory available to a browser.
4. Copy one of the UTM test links at the bottom into an (incognito?) browser window.
5. Confirm the [ID value](https://en.wikipedia.org/wiki/WooCommerce#:~:text=WordPress.%20Launched%20on-,September%2027%2C%202011,-%2C%5B3%5D) is displayed correctly in First visit and Current visit:
    ![image](https://github.com/woocommerce/sourcebuster-js/assets/228780/0cbe764d-6948-4e68-9877-42ed110c6b87)
6. Update the `utm_id` value and confirm that the Current visit ID value changes.
